frontend/projects/valtimo/components/src/lib/components/case-tags-selector/case-tags-selector.component.scss

```
@@ -23,6 +23,14 @@
   margin-top: 8px;
 }
 
+::ng-deep .case-tag-selector .cds--list-box__selection--multi {
```

Stylelint (17.4.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)